nservicebus/testing/index_content_testing_[6,).partial.md

```diff
@@ -18,14 +40,29 @@ The following unit test checks that the total amount has the discount applied:
 
 snippet: SagaTest
 
+
 ## Testing a behavior
 
+[Message pipeline behaviors](/nservicebus/pipeline/manipulate-with-behaviors.md) also can be tested, but using a different testable context objects. Each of the [various pipeline stages](/nservicebus/pipeline/steps-stages-connectors.md) uses a specific interface for its context, and each context interface has a testable implementation following a name convention.
+
+To determine the testable context for a behavior context, replace the `I` at the beginning of the interface name with `Testable`.
+
+For example: 
+
+* A behavior using `IIncomingLogicalMessageContext` can be tested using `TestableIncomingLogicalMessageContext`.
+* A behavior using `IInvokeHandlerContext` can be tested using `TestableInvokeHandlerContext`.
+* etc.
+
+Refer to the [pipeline stages document](/nservicebus/pipeline/steps-stages-connectors.md) for a complete list of the available behavior contexts.
+
+Each of these testable types contains properties similar to `TestableMessageHandlerContext` that can be used to assert that a behavior is working as designed.
```

Comment:
"similar to `TestableMessageHandlerContext`"

My first inclination is drop this, but if you want to keep it, it should be more like "similar to `TestableMessageHandlerContext`'s properties"